### ISSUE TITLE
Paste as plain text not working with expected shortcut "Ctrl + Shift + V"

### ISSUE THREAD

**Issue by uncertainquark** [ONLYOFFICE/DesktopEditors#165]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Pasting copied content as plain text is not working using the universally expected shortcut "Ctrl + Shift + V"

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Try pasting this formatted text using ""Ctrl + Shift + V":
**_I came across related articles on your website and want to convey interest in contributing to the same._**

**What is the expected behavior?**
It doesn't paste as plain text i.e like this:
I came across related articles on your website and want to convey interest in contributing to the same.

![screenshot from 2019-01-15 14-44-39](https://user-images.githubusercontent.com/22585732/51170529-b2002c80-18d4-11e9-9467-5fa2485608a3.png)

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
v5.2.4

**Comment by Rita-Bubnova** [comment]
Hello, @uncertainquark.
We try to support keyboard shortcuts used in MS Office.
Shortcuts
Ctrl + Shift + C and Ctrl + Shift + V are used to copy and paste formatting, respectively.

Link to Keyboard shortcuts for Microsoft Word on Windows:
https://support.office.com/en-us/article/keyboard-shortcuts-for-microsoft-word-on-windows-95ef89dd-7142-4b50-afb2-f762f663ceb2?ui=en-US&rs=en-US&ad=US

**Comment by uncertainquark** [comment]
Okay. However, it is not working for me as intended either. See screenshot below.
![screenshot from 2019-01-16 10-13-02](https://user-images.githubusercontent.com/22585732/51226872-808c6d00-1977-11e9-9e7d-25a787cf3347.png)

Attaching the document as well for you'll to reproduce: [plain-text.docx](https://github.com/ONLYOFFICE/DesktopEditors/files/2762670/plain-text.docx)

**Comment by ShockwaveNN** [comment]
@uncertainquark to use `control+shift+v` you should first copy style by pressing `ctrl+shift+c`
If you do  `control+shift+v` in new document - it will do nothing

**Comment by uncertainquark** [comment]
Hi, I tried Ctrl+Shift+C and then Ctrl+Shift+V and it still doesn't work. And Ctrl+V pastes it as shown above, neither pasting the original formatting, nor pasting it as plain text. Sorry for not being able to convey appropriately, can you help identify the issue?

**Comment by ShockwaveNN** [comment]
@uncertainquark yep, can you describe every step which you do? I don't think there is known issues with this hotkeys

**Comment by uncertainquark** [comment]
To add: The way Google Docs does is it uses "Ctrl+C" to copy the text (formatted) and pastes in the following ways:
1. Ctrl+V = Paste formatted
2. Ctrl+Shift+V =  Paste as plain text

I've found it to be much simpler because only one shortcut (Ctrl+C) copies the text to the clipboard (and is the default one that everyone remembers) and the decision to paste the text formatted/plain need to be taken only once, at the second step. What do you guys think?

**Comment by ShockwaveNN** [comment]
@uncertainquark Most of our clients moved from Microsoft Office to our product, not from Google Docs, so we try to use shortcuts most convenient to Microsoft Office users

**Comment by uncertainquark** [comment]
> @uncertainquark yep, can you describe every step which you do? I don't think there is known issues with this hotkeys

1. Copy text from an article/email in a browser tab.
2. Paste using Ctrl+V: 
It pastes formatted text, but the wrong format (as shown in screenshots above). Moreover, it doesn't paste as plain text as intended by the shortcut, if I am understanding correctly.
3. Ctrl+Shift+C open developer console in Firefox so I tried using the shortcut in the document itself with the formatted text.
4. Pasting the formatted text as Ctrl+Shift+V doesn't do anything.

**Comment by ShockwaveNN** [comment]
> the wrong format (as shown in screenshots above)

Ok, now I understand your problem, I think it may be separate issue (not related to hotkeys, but related to email client and list of installed fonts) I think we should back to it after we resolve hotkeys problems.

> Moreover, it doesn't paste as plain text as intended by the shortcut, if I am understanding correctly.

No, ctrl+v should insert formatted text, not plain. It was designed that way.

> Ctrl+Shift+C open developer console in Firefox

You can only copy-paste style from our editor. So you should paste text in DesktopEditors, select it and press `Ctrl+Shift+C`, and after that you can paste that style on other text in DesktopEditors

**Comment by uncertainquark** [comment]
> > Moreover, it doesn't paste as plain text as intended by the shortcut, if I am understanding correctly.
> 
> No, ctrl+v should insert formatted text, not plain. It was designed that way.

I want to be able to paste the copied clipboard content as plain text with a shortcut. If "Ctrl+V" and "Ctrl+Shift+V" both paste formatted content, how to paste as plain text using a shortcut?

**Comment by ShockwaveNN** [comment]
> If "Ctrl+V" and "Ctrl+Shift+V" both paste formatted content,

"Ctrl+Shift+V" both paste formatted content is not to paste content with style, but only paste style, it will not add any text data to document

> how to paste as plain text using a shortcut?

Theres is no such shortcut, you can only use paste shortcut and clear text style after that by pressing button on top toolbar
